installer/common/shared.sh

```diff
@@ -57,7 +57,7 @@ echo "The tiller namespace is '$TILLER_NAMESPACE'"
 
 # Cloudflow operator version
 export operatorImageName="lightbend/cloudflow-operator"
-export operatorImageTag="1.3.0"
+export operatorImageTag="89-f8a4ddc"
```

Comment:
@skonto don't we need to automate this with the release of a new operator version ? May be in a separate PR .. LGTM this ..

Reply:
This is snapshot version for master. It is not a release.

Reply:
@debasishg and @skonto I think this will be the very first step we will need to pick up when we want to start tackling daily integration builds, e.g. the first step is to build (named) snapshots of everything  and then point internal dependencies to the artifacts just built until we have an entire self-compliant "release" of master.

One important question is whether such a daily integration build should just always update git to point everything to verified CI builds. I don't see any major issues with that. WDYT?

Reply:
If nothing changes we need to avoid unnecessary commits. In general yes that would be the ideal case but if something is broken we need also to be able to avoid automatic triggering until we do a fix we missed (we cannot have 100% integration test coverage). So maybe manual trigger after each PR is more conservative but suitable for the time being.